### ISSUE TITLE
Discord link is broken (website)

### ISSUE THREAD

**Issue by Dryhb** [LAION-AI/Open-Assistant#1233]
The Discord link at the bottom of the page is broken (exprired invite)

**Comment by swaptr** [comment]
Both the discord links on the webpage seem to work for me. Can you post what issue you are facing?

**Comment by Lyu23** [comment]
> Both the discord links on the webpage seem to work for me. Can you post what issue you are facing?

1

**Comment by Lyu23** [comment]
> Both the discord links on the webpage seem to work for me. Can you post what issue you are facing?

1

**Comment by swaptr** [comment]
Thanks for updating the issue, both the links i.e. the one in Call to action component on the home page and the footer point to the same URL `https://ykilcher.com/open-assistant-discord`. I am not sure why you are facing this issue.

**Comment by andreaskoepf** [comment]
As @swaptr mentioned, the official OA discord invite-link is: https://ykilcher.com/open-assistant-discord
@swaptr Please let us know where you found the expired link.

**Comment by andreaskoepf** [comment]
2nd report: https://github.com/LAION-AI/Open-Assistant/issues/465#issuecomment-1416909932

**Comment by Dryhb** [comment]
Ok that seems really strange, on phone the link works and I checked on Pc and phone the same link is opened when clicking on the discord text. Furthermore, I tried on google chrome chrome and the link works, the discord invite "https://discord.com/invite/H769HxZyb5" shows up as the same in my original browser but somehow it still shows (on Brave browser precisely) as invalid invite even tho it is the exact same link. Probably an issue on my end. When I reload the page it briefly show the discord name then changes  to invalid invite. Very strange

So it's probably on my end.
